cfgov/v1/models/browse_page.py

```diff
@@ -0,0 +1,54 @@
+from wagtail.wagtailcore.fields import StreamField
+from wagtail.wagtailadmin.edit_handlers import TabbedInterface, ObjectList, \
+    StreamFieldPanel
+from wagtail.wagtailcore import blocks
+from wagtail.wagtailcore.models import PAGE_TEMPLATE_VAR
+
+from .base import CFGOVPage
+from . import molecules
+from . import organisms
+
+
+class BrowsePage(CFGOVPage):
+    header = StreamField([
+        ('text_introduction', molecules.TextIntroduction()),
```

Comment:
Shouldn't there be a Featured Content molecule here too? Just comparing from the wiki image.